document-generation/eval-plan-document.ts

```diff
@@ -0,0 +1,28 @@
+import { logger } from "../utils/logging";
+import createReport from "docx-templates";
+import { EvaluationPlan } from "../models/document-generation";
+import { ApiBase64SuccessResponse, SuccessStatusCode } from "../utils/response";
+import { getFundingDocInfo } from "./utils/utils";
+
+export async function generateEvalPlanDocument(
+  template: Buffer,
+  payload: EvaluationPlan
+): Promise<ApiBase64SuccessResponse> {
+  const report = Buffer.from(
+    await createReport({
+      template,
+      data: {
+        ...payload,
+      },
```

Comment:
Do you plan on updating other values inside the payload? If not then you probably don't need to spread the payload here.

Reply:
Might be worth it to keep the payload this way if there are future changes.